charts/ndb-operator/templates/mutatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: ndb-operator-system/ndb-operator-serving-cert
```

Comment:
Template?

Reply:
I am tending towards holding off templating the webhook-related internal stuff (as long as that is not causing issues with helm install).